src/components/DropdownMenu.spec.tsx

```diff
@@ -17,70 +17,75 @@ describe('DropdownMenu', () => {
     </DropdownMenu>
   );
 
-  it('should open on a button click and close on another button click', () => {
-    const component = renderer.create(<TestMenu variant='light'/>);
-    expect(component.toJSON()).toMatchSnapshot();
+  // Need to isolate tests from the 2 libraries in these blocks otherwise we get warnings about using the wrong act()
+  describe('@testing-library/react', () => {
```

Comment:
Interesting, I wouldn't expect a `describe` to provide more isolation than an `it`, unless using the module name as the describe name is doing something special here (I don't think so?)

Reply:
yea i'm also not sure how the describe is helping in this case